File: single-language-analysis/dataset-files/java/TestSmell049.java
// Original URL: https://github.com/AJohnstone2007/referenceLanguageCorpora/blob/2e8331957fa4c61d7979be7b54792a23fc8be724/languages/java/corpus/javaFX/cws/SkinCleanupTest.java#L68-L77

public class TestSmell049 {
@Ignore("JDK-8277000")
@Test
public void testTreeTableRowFixedCellSizeListener() {
TreeTableView<Person> tableView = createPersonTreeTable(false);
showControl(tableView, true);
TreeTableRow<?> tableRow = (TreeTableRow<?>) getCell(tableView, 1);
TreeTableRowSkin<?> rowSkin = (TreeTableRowSkin<?>) tableRow.getSkin();
assertNull("row skin must not have listener to fixedCellSize",
unregisterChangeListeners(rowSkin, tableView.fixedCellSizeProperty()));
}
}

File: single-language-analysis/dataset-files/python/test_smell_049.py
import pytest

class TestSkinCleanup:
    @pytest.mark.skip(reason="JDK-8277000")
    def test_smell_049(self):
        table_view = self.create_person_tree_table(False)
        self.show_control(table_view, True)
        table_row = self.get_cell(table_view, 1)
        row_skin = table_row.get_skin()
        assert row_skin.unregister_change_listeners(table_view.fixed_cell_size_property()) is None